.env

```diff
@@ -0,0 +1,9 @@
+# The contents of this file MUST be altered during deployment, to ensure that your application's passwords are entire secured
```

Comment:
Perfect for now; this is the same policy I advised Reetta to use for the MillMaps repo: all sensitive information in a single location where it can be easily kept safe.